chore: add docs for HTTP email delivery (#1501)
ory/docs: docs/kratos/self-hosted/03_mail-courier-http.mdx

---
id: email-http
title: HTTP based email delivery
sidebar_label: Sending emails via HTTP API
---

> For sending emails by using an external mail provider instead of a local SMTP server, Ory Kratos can use an HTTP API (such as

Reviewer: ```suggestion
To send emails using an external mail provider instead of a local SMTP server, Ory Kratos can use an HTTP API (such as
```

Mailchimp, your local mail sender, or your own microservice). Request method, headers, body, and content-type are fully
configurable using the options below.

## Configuration

> By default, Ory Kratos doesn't use API calls to send mail. To enable it please set the `delivery_strategy` configuration key to

Reviewer: ```suggestion
By default, Ory Kratos doesn't use API calls to send mail. To enable it set the `delivery_strategy` configuration key to
```
